ios/FluentUI/Button/Button.swift

```diff
@@ -110,7 +110,7 @@ open class Button: UIButton, Shadowable, TokenizedControlInternal {
         layer.cornerCurve = .continuous
 
         titleLabel?.font = tokenSet[.titleFont].uiFont
-        titleLabel?.adjustsFontForContentSizeCategory = true
+        titleLabel?.adjustsFontForContentSizeCategory = !style.isFloating
```

Comment:
We should also set `shouldScale` to `false` in the font tokens. And you might need to add something to the title transformer in the `if #available(iOS 15, *)` check just below here.

Reply:
Ah, I expanded the preview, and I think as long as the font token says not to scale you shouldn't need to update the title transformer.